guides/oidc/oidc.md

```diff
@@ -0,0 +1,116 @@
+# OpenID Connect
+
+The OpenID Connect (OIDC) feature allows third-party applications to verify the identity of the end-user and to obtain basic user profile information. Instead of managing usernames and passwords directly **trento-web console will rely on an identity provider (IDP) to authenticate users**.
```

Comment:
I wouldn't call this OIDC feature, this is an integration.
Maybe `Trento is able to integrate with OpenID Connect protocol identity providers to authenticate users accessing the console. In the other hand, authorization of different abilities is still managed in Trento entirely, so only some basic user information is retrieved from the external platform.`

I don't know, something talking about integration.